recipes/chromsize/build.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
+
```

Comment:
You don't need this anymore, please just add the rust conpliner into the build section